api/src/http/routes/profile-activate.ts

```diff
@@ -0,0 +1,17 @@
+import { FastifyInstance } from "fastify";
+import HttpError from "./utils/http-error";
+import { toggleProfile } from "./utils/toggle-profile";
+
+export async function updateProfileActivate(server: FastifyInstance) {
+    server.patch("/profile/activate/:profileId", async (request, reply) => {
+        try {
+            const res = await toggleProfile(request, true);
+            reply.status(200).send(res);
```

Comment:
Coloca um `return` em cada `reply` pra garantir que nada vai ser executado depois de um `reply` por qualquer motivo.

```suggestion
            return reply.status(200).send(res);
```